diff --git a/node_modules/react-native-camera/src/RNCamera.js b/node_modules/react-native-camera/src/RNCamera.js
index b7a271a..7fdfa5a 100644
--- a/node_modules/react-native-camera/src/RNCamera.js
+++ b/node_modules/react-native-camera/src/RNCamera.js
@@ -1,19 +1,18 @@
 // @flow
-import React from 'react';
+import { ViewPropTypes } from 'deprecated-react-native-prop-types';
 import PropTypes from 'prop-types';
+import React from 'react';
 import {
-  findNodeHandle,
-  Platform,
-  NativeModules,
-  ViewPropTypes,
-  requireNativeComponent,
-  View,
   ActivityIndicator,
-  Text,
-  StyleSheet,
+  NativeModules,
   PermissionsAndroid,
+  Platform,
+  StyleSheet,
+  Text,
+  View,
+  findNodeHandle,
+  requireNativeComponent,
 } from 'react-native';
-
 import type { FaceFeature } from './FaceDetector';
 
 const Rationale = PropTypes.shape({
